site/src/AppRouter.tsx

```diff
@@ -12,21 +12,23 @@ import { XServiceContext } from "xServices/StateContext"
 import { AuthAndFrame } from "./components/AuthAndFrame/AuthAndFrame"
 import { RequireAuth } from "./components/RequireAuth/RequireAuth"
 import { SettingsLayout } from "./components/SettingsLayout/SettingsLayout"
-import { IndexPage } from "./pages"
-import { NotFoundPage } from "./pages/404Page/404Page"
-import { CliAuthenticationPage } from "./pages/CliAuthPage/CliAuthPage"
-import { HealthzPage } from "./pages/HealthzPage/HealthzPage"
-import { LoginPage } from "./pages/LoginPage/LoginPage"
-import { TemplatesPage } from "./pages/TemplatesPage/TemplatesPage"
-import { AccountPage } from "./pages/UserSettingsPage/AccountPage/AccountPage"
-import { SecurityPage } from "./pages/UserSettingsPage/SecurityPage/SecurityPage"
-import { SSHKeysPage } from "./pages/UserSettingsPage/SSHKeysPage/SSHKeysPage"
-import { CreateUserPage } from "./pages/UsersPage/CreateUserPage/CreateUserPage"
-import { UsersPage } from "./pages/UsersPage/UsersPage"
-import { WorkspaceBuildPage } from "./pages/WorkspaceBuildPage/WorkspaceBuildPage"
-import { WorkspacePage } from "./pages/WorkspacePage/WorkspacePage"
-import { WorkspaceSchedulePage } from "./pages/WorkspaceSchedulePage/WorkspaceSchedulePage"
 
+const IndexPage = lazy(() => import("./pages"))
```

Comment:
Remind me again, why are we lazy loading? Speeds up things or something?

Reply:
Yes, so the pages are not loaded in the main bundle but I'm going to change this and don't lazy load the main pages - the ones that are in the nav.